src/java/edu/ucsd/library/dams/util/EncryptedStreamNameModule.java

```diff
@@ -319,12 +312,15 @@ private boolean startsWithDigit(String s) {
 	  
 	public static void main( String[] args ) throws Exception
 	{
-		String[] parts = args[0].split(",");
+		String encryptedtext = "kf8ih30qyrw39rpb,wSTL7GcDUZd1ic1qLwKA3ubn9Amh4rbr5NQymDdDvwT-GZW-KkzsLJfz4dd8KzwFHG5xdrqslOyaY7_m2Y60wfkJNDsVw2RbpSCxOCsX6FA=";
+		String[] parts = encryptedtext.split(",");
 		String nonce = parts[0];
 		String ciphertext = parts[1];
 		EncryptedStreamNameModule module = new EncryptedStreamNameModule();
 		module.keyFile = "streaming.key";
 		String plaintext = module.decrypt( nonce, ciphertext );
 		System.out.println("decrypted: " + plaintext);
+		String expected = "3j333268h d504rk75p 34cddd4138eeb3686bdb095b9a26ac077919b0f0 mp3 127.0.0.1";
+		Assert.assertEquals("Descripted text need to be matched", expected, plaintext);
```

Comment:
Thanks for adding a legitimate test 👏 Is it a lot of work to create a legitimate test file for this instead of having it in `main`? If so, we could chat about whether it's worth it to do so. If not, it might be nice since we could potentially rig up CI for this repo.